ui/scss/component/_file-thumbnail.scss

```diff
@@ -1,13 +1,13 @@
 .file-thumbnail__viewed-bar {
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  width: 100%;
-  height: 5px;
-  background-color: var(--color-gray-7);
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 10px; // UPDATE: Changed thumbnail viewed bar height to 10px
```

Comment:
Not sure about this.

Reply:
Same. Personal styling that I forgot to remove before the pull request.